src/app/GlobalRedux/Features/MarketTableSlice/index.ts

```diff
@@ -1,15 +1,14 @@
 `use client`;
 
 import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
-import { Coin } from "../../../../../interfaces";
 
 export const fetchCoins = createAsyncThunk(
   "coins/getCoins",
   async (currency: string, thunkApi) => {
     const COIN_URL = `https://api.coingecko.com/api/v3/coins/markets?vs_currency=${currency}&order=market_cap_desc&per_page=20&page=1&sparkline=true&price_change_percentage=1h%2C24h%2C7d&locale=en`;
     const response = await fetch(COIN_URL);
     const data = await response.json();
-    console.log(data);
+    console.log("fetch requested:", data);
```

Comment:
no

Reply:
Gone. This was from creating a branch off PR 37 instead of master.